index.html

```diff
@@ -210,6 +217,23 @@ <h2>Attributes</h2>
                     reflected in the <a>[[\PlayoutDelayHint]]</a> internal slot.
                   </p>
                 </li>
+                <li>
+                  <p>Modifying the delay of the underlying system SHOULD affect
+                  the internal audio or video buffering gradually in order not
+                  to hurt user experience. Audio samples or video frames SHOULD
+                  be accelerated or decelerated before playout, similarly to how
+                  it is done for
+                  <a href="https://tools.ietf.org/html/rfc5888#section-7">
+                  audio/video synchronization</a> or in response to congestion
+                  control.</p>
```

Comment:
Can we put any limits on how long implemenations are allowed to take here, to make this testable?

Reply:
Overall time needed to achieve depends on the network conditions. Network delays and packet's drops can heavily influence overall time between receiving and playout. 

Theoretically, we can specify  how fast it is allowed to stretch (deaccelerate) and shorten (accelerate) audio and video as these parameters are essentially putting margins on "don't hurt User Experience".  This would allow to specify how fast we can achieve target in the case of "perfect network conditions" which in practice is the localhost connection.

But at least in libwebrtc implementation those parameters are actively adjusted through experimentation and most importantly in audio case they are tied up with the type of the audio. For example you can accelerate almost instantly if your audio frame contains lots of silence/minor background noise. If it contains speech then speech related traits play major role such as peeks, pace, timber, etc. Essentially, shape of the waveform. Thus, even for localhost testing you would get different results depending on audio being used.

I want to emphasize that this testing issue would be present independently from the API path chosen.

Reply:
We can try to provide maximum time beyond which should be considered a bug in "perfect network" conditions. It should give some reference for the vendors. But I would advise against it. I can measure the Chrome speed with some constant speech feed and take x2.0 of that time to estimate the maximum allowed margin.

Reply:
Let's follow up on this in #11.